docs/periodic.adoc

```diff
@@ -48,7 +48,7 @@ So if you were to add an `:Actor` label to several million `:Person` nodes, you
 ----
 CALL apoc.periodic.iterate(
 "MATCH (p:Person) WHERE (p)-[:ACTED_IN]->() RETURN p",
-"SET p:Actor", {batchSize:10000, parallel:true})
+"WITH {p} AS p SET p:Actor", {batchSize:10000, parallel:true})
```

Comment:
you don't need that, it's handled automatically.

Reply:
Is there any differences between 3.0.x and later versions? My query didn't work until I did this and would say that `Variable ``p`` is not defined` etc

Reply:
If so I should maybe PR this against the documentation for the older versions but not against later then

Reply:
Looks like you automatically do this in https://github.com/neo4j-contrib/neo4j-apoc-procedures/commit/35d83dfa50e547c5b40c64877172a7bd361d7e53#diff-f240d1755aedbb2670af0b9ed9776fc9 (3.2?) and remove the documentation. But this information has been removed from the 3.0.x docs:  https://neo4j-contrib.github.io/neo4j-apoc-procedures/index30.html#_apoc_periodic_iterate